lib/ui/uikit/src/components/PlDropdown/PlDropdown.vue

```diff
@@ -59,6 +59,10 @@ const props = withDefaults(
      * Error message displayed below the dropdown (optional)
      */
     error?: unknown;
+    /**
+     * If false there is only red border with non-empty error field, without a message
+     */
+    showErrorMessage?: boolean;
```

Comment:
I don't think that you should flag that control another. MB `errorStatus: true`|`status: 'error'|'idle'` will be better?